### PR TITLE
Add form to mobile version.

### PR DESCRIPTION
**Pull request by gfrancoarq**
### I've added the contact form to the mobile version of my portfolio. It includes content and styling.
* These are the mobile website sections where I will list my portfolio projects and skills.
* These changes complete the "Portfolio: add a contact form and finish the mobile version" activity

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -348,6 +348,72 @@ <h4>Skills</h4>
...
+            class="grey-icon"
+          />
+        </li>
+      </ul>
+
+      <p class="email description">gfrancoarquitecto@gmail.com</p>
+    </div>
+
+    <!-- Project 3 Portfolio: add a contact form and finish the mobile version END -->
+
```
Comment (ichala) [inline index.html:416]: - [ ] kindly remove any unnecessary comments to make your code clean

File: index.html [hunk 2]
```diff
@@ -348,6 +348,72 @@ <h4>Skills</h4>
...
 
+    <!-- Project 3 Portfolio: add a contact form and finish the mobile version START -->
+
+    <div class="form-container">
+      <h2 class="recent-works container-title">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+
+      <form action="/action_page.php" method="post">
```
Comment (ichala) [inline index.html:359]: - [ ] the action method should be a [formspree](https://formspree.io/html) link `https://formspree.io/f/{form_id}` you will get your own `form_id` once you create an account

File: index.html [hunk 3]
```diff
@@ -348,6 +348,72 @@ <h4>Skills</h4>
...
+            type="email"
+            name="email-addres"
+            placeholder="Email address"
+          />
+
+          <textarea
+            class="form-text"
+            name="message"
+            placeholder="Enter text here"
+          ></textarea>
```
Comment (ichala) [inline index.html:379]: - [ ] kindly add all the required HTML validations

1. Name input should have `maxlength="30"` attribute.
1. textarea input should have `maxlength="500"` attribute.
1. All input fields have the `required` attribute to activate the validation on submit.

File: index.html [hunk 4]
```diff
@@ -348,6 +348,72 @@ <h4>Skills</h4>
 
     <!-- Project 2 Portfolio: finish mobile version END -->
 
+    <!-- Project 3 Portfolio: add a contact form and finish the mobile version START -->
+
+    <div class="form-container">
```
Comment (ichala) [inline index.html:353]: - [ ] the form design looks awesome 👏 kindly add the missing background image like shown in figma

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @gfrancoarq , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linters errors✅ 
- Good Design ✅ 

 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — APPROVED** [review]
Hi @gfrancoarq,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
No linters errors✅
Good Design ✅

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**